scripts/e2e_test.sh

```diff
@@ -1,36 +1,194 @@
 #!/bin/bash
 
-set -x
+set -eux
 
-# Specify the command-line flags for the fuzzing process
+# ====== CONFIGURATION ======
+
+# Temporary Variables
+readonly PROJECT_SRC_PATH="https://github.com/go-continuous-fuzz/go-fuzzing-example.git"
+readonly FUZZ_TIME="3m"
+readonly MAKE_TIMEOUT="5m"
+
+# Use test workspace directory
+readonly TEST_WORKDIR="$(pwd)/test-workdir"
```

Comment:
This script should be runnable from any CWD.  Any temp test files should be created in a  `mktemp` directory to avoid name collisions.